README.md

```diff
@@ -11,3 +11,43 @@ The purpose of this repository is to consolidate all the community builders for
 
 # Further reading
 [Angular CLI 6 under the hood - builders demystified](https://medium.com/@meltedspark/angular-cli-6-under-the-hood-builders-demystified-f0690ebcf01)
+
+## Troubleshooting
+
+Please find below a selection of potential issues you might face when using this builder. Refer to [jest-preset-angular Troubleshooting](https://github.com/thymikee/jest-preset-angular) for angular jest preset specific issues
```

Comment:
Please use _jest-preset-angular_ instead of angular jest preset. It's more readable and has more sense